docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+data:
+    image: busybox
+    volumes:
+        - ../:/var/www/app
+
+nginx:
+    image: yappabe/nginx
```

Comment:
You can add all configuration files, Dockerfiles and other stuff in a `docker` dir at the root of the project. (same as `vagrant`).

For developers, it will be straight-forward since he doesn't have to do anything to make it work.